### PR TITLE
Journalist Can See Page of Articles Reviewed and Status

### PR DESCRIPTION
**Pull request by rupertlion**
## PT Link

https://www.pivotaltracker.com/story/show/161365886

## Included in PR

* Add submitted articles list on dashboard view when logged in to account
* Add link to article creation/edit page for articles with comments
* Add ability to update article and resubmit for approval
* NOTE: Task - "Add ability to update article and resubmit for approval" has been moved to a separate PR due to size of task. Please see separate PR "Journalist can Edit and Resubmit Article Marked as For_Revision by Editor Review"

image

### CODE REVIEW

**Review by RhonGabriel — APPROVED** [review]

**Review by angiecstn — APPROVED** [review]

File: app/controllers/admin/dashboard_controller.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 class Admin::DashboardController < Admin::AdminController
-  def index; end
-end
+  def index
+    @articles = Article.where.not(status: 0)
```
Comment (tochman) [inline app/controllers/admin/dashboard_controller.rb]:
```
@articles = !Article.pending
```

Comment (tochman) [reply app/controllers/admin/dashboard_controller.rb]:
```
Article.where.not(status: :pending)
```

Comment (tochman) [reply app/controllers/admin/dashboard_controller.rb]:
```
Article.not.pending
```
This will probably not fly

**Review by tochman — APPROVED** [review]

File: app/controllers/admin/dashboard_controller.rb (hunk 1 above)
Comment (rupertlion) [reply app/controllers/admin/dashboard_controller.rb]: the only one that works is this one 
```
Article.where.not(status: :pending)
```
so will change to that

**Pull request merged**